source/pug/templates/base-for-django.pug

```diff
@@ -8,6 +8,10 @@ block masterParams
   - let isMezzanine = true
   - let pageTitle = `{{ page.meta_title }}`;
 
+block metaTags
+  | {% load wagtailmetadata_tags %}
+  | {% meta_tags %}
```

Comment:
this should now go in the modular_page.html template in the wagtailpages app, as:
```
{% load wagtail..... wagtailmetadata_tags %}

{% block socialMetadata %}
    {% meta_tags %}
{% endblock %}
```